.autocopr/podman.spec

```diff
@@ -0,0 +1,178 @@
+%global with_debug 0
+
+%if 0%{?with_debug}
+%global _find_debuginfo_dwz_opts %{nil}
+%global _dwz_low_mem_die_limit 0
+%else
+%global debug_package %{nil}
+%endif
+
+%global provider github
+%global provider_tld com
+%global project containers
+%global repo %{name}
+# https://github.com/containers/%%{name}
+%global import_path %{provider}.%{provider_tld}/%{project}/%{repo}
+%global git0 https://%{import_path}
+
+Name: podman
+Epoch: 100
+Version: 4
+%define build_timestamp %{lua: print(os.date("%Y%m%d"))}
```

Comment:
I don't know COPR conventions but my immediate thought here is, will there ever be an occasion in which you need to build twice in one day?

Reply:
err, yes good idea, let me include the timestamp as well. I just copypasted from the blog post.